ports/rp2/boards/ARDUINO_NANO_RP2040_CONNECT/manifest.py

```diff
@@ -3,9 +3,12 @@
 # Networking
 require("webrepl")
 require("urequests")
+require("ntptime")
+require("kpn-senml")
```

Comment:
I suggest removing this kpn-senml line for now, then this PR can be unblocked and merged.

Reply:
I removed `kpn-senml` and squashed the commits. I also added `logging` to manifests, it's limited but this way when it gets updated in `micropython-lib` it's going to be enabled.